src/runtime/c/lib/util/message.c

```diff
@@ -1,84 +1,86 @@
-/* 
- * Copyright (c) 2017-2018, SyLabs, Inc. All rights reserved.
+/*
+ * Copyright (c) 2018, SyLabs, Inc. All rights reserved.
  * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
```

Comment:
I think (@gmkurtzer or @jmstover correct me if I'm wrong) that we no longer need to include the SingularityWare copyright line, as when Sylabs was created we transferred all IP from SingularityWare -> Sylabs. So we should remove the SingularityWare line, and update the Sylabs line to be:

`Copyright (c) 2017-2018, Sylabs Inc. All rights reserved.`